### ISSUE TITLE
Error setting up minion nodes with CoreOS 695.0

### ISSUE THREAD

**Issue by yoanisgil** [pires/kubernetes-vagrant-coreos-cluster#105]
Hi,

I am trying to setup my cluster with CoreOS 695.0 (latest release in the alpha channel) and I get this from  journalctl:

```
May 29 00:23:41 node-01 etcd2[1185]: 2015/05/29 00:23:41 error verifying flags, -advertise-client-urls is required when -listen-client-urls is set explicitly. See 'etcd -help'.
May 29 00:23:41 node-01 etcd2[1185]: 2015/05/29 00:23:41 When listening on specific address(es), this etcd process must advertise accessible url(s) to each connected client.
May 29 00:23:41 node-01 systemd[1]: etcd2.service: main process exited, code=exited, status=1/FAILURE
```

Seems like the etcd command line has changed a bit. What's the recommend CoreOS version to go with?

Thanks,

Yoanis.

**Comment by yoanisgil** [comment]
@pires  This patch:

```
diff --git a/node.yaml b/node.yaml
index 02e8487..20e0c63 100644
--- a/node.yaml
+++ b/node.yaml
@@ -27,6 +27,7 @@ coreos:
   etcd2:
     name: __NAME__
     listen-client-urls: http://0.0.0.0:2379,http://0.0.0.0:4001
+    advertise-client-urls: http://0.0.0.0:2379,http://0.0.0.0:4001
     initial-cluster: __ETCD_SEED_CLUSTER__
     proxy: on
   fleet:
```

solves the issue for me. Let me know if you want me to create a PR and I would gladly do, though I am not entirely sure if this is backwards compatible with previous CoreOS versions. If not what could be done to support CoreOS >= 695.0 and CoreOS < 695.0?

Bests,

Yoanis.

**Comment by pires** [comment]
@yoanisgil please do.

**Comment by AntonioMeireles** [comment]
@yoanisgil fwiw.afaict the patch is backwards compatible.

**Comment by pires** [comment]
Yes it is.

**Comment by yoanisgil** [comment]
PR is done ;) .. See here: https://github.com/pires/kubernetes-vagrant-coreos-cluster/pull/106

**Comment by pires** [comment]
Fixed by #106.
